lib/testmap.py

```diff
@@ -48,6 +48,7 @@
         ],
         # These can be triggered manually with bots/tests-trigger
         '_manual': [
+            'fedora-36',
```

Comment:
I recommend a separate PR which adds fedora-36 to *all* our projects (to `_manual`), so that we can enable them one by one.